scripts/entrypoint.sh

```diff
@@ -13,6 +15,17 @@ if [[ -e /test ]]; then
     ls -la /test
 fi
 
-entrypoint.py
+WRITABLE_PATH=/builds/worker/tmp
+# we're not using livelog yet, set key to something so g-w will start
+export LIVELOG_SECRET='not_a_key'
+
 cd $HOME
-exec taskcluster-worker work /builds/taskcluster/${TC_WORKER_CONF}.yml
+generic-worker new-ed25519-keypair --file $WRITABLE_PATH/ed25519_private_key
+generic-worker new-openpgp-keypair --file $WRITABLE_PATH/openpgp_private_key
+envsubst < /builds/generic-worker/generic-worker.yml.template > $WRITABLE_PATH/generic-worker.yml
+
+# user.Current requires cgo, but cross-compilation doesn't enable cgo
+# this works around the issue (see https://github.com/golang/go/issues/14625
+# and https://github.com/ksonnet/ksonnet/issues/298)
+export USER=root
+exec generic-worker run --config $WRITABLE_PATH/generic-worker.yml
```

Comment:
It is not clear to me how this will work. The script.py in the previous taskcluster-worker approach would download the test script from the payload context and run that to actually start the test.

You may or may not want to keep some of the logic from the script.py file. If you do, I would update it and have it call the generic-worker. Though I must admit I'm not familiar with the generic-worker approach.

Let's talk about this in more detail.